tests/lib/pkgdb.sh

```diff
@@ -893,3 +893,75 @@ distro_upgrade() {
             ;;
     esac
 }
+
+distro_list_packages() {
+    where="$1"
+
+    case "$SPREAD_SYSTEM" in
+        fedora-*|centos-*|amazon-linux-*|opensuse-*)
+            rpm -qa | sort > "$where"
+            ;;
+        ubuntu-*|debian-*)
+            dpkg --get-selections > "$where"
```

Comment:
Unlike the rest, this prints more than the package name.

Reply:
Yeah, that's how selections work. OTOH, it's handled directly by `dpkg --set-selections`.